Source/PCH/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 add_library(use_pch INTERFACE)
 
 # Uncomment this return to disable PCH
-#return()
+return()
```

Comment:
Unintentional?

Reply:
Yup. Noticed immediately after I made the PR